dashstack/styles/devstyle.css

```diff
@@ -0,0 +1,61 @@
+.dash-logo {
+  color: var(--primary-dark-blue);
+}
+.container {
+  position: relative;
+  width: 100%;
+}
+.sidebar {
+  position: fixed;
+  width: 130px;
+  height: 100%;
```

Comment:
I'm not sure about the outcome because you have not provided any markup
but I think `100dvh` would serve your needs better